src/Core/src/Credentials/PsrCacheProvider.php

```diff
@@ -27,6 +28,18 @@ public function __construct(CredentialProvider $decorated, CacheItemPoolInterfac
     }
 
     public function getCredentials(Configuration $configuration): ?Credentials
+    {
+        try {
+            return $this->getFromCache($configuration);
+        } catch (CacheException $e) {
```

Comment:
This behaviour is already implemented inside Symfony Cache, if it was fine to rely on `\Symfony\Contracts\Cache\CacheInterface` instead, then the logic doesn't need to be repeated. But I assume it is intentional to rely on the PSR interface instead?

Reply:
For reference: https://github.com/symfony/cache/blob/5.x/Traits/AbstractAdapterTrait.php#L110

Reply:
Yeah. You are correct. For a Symfony PSR6 cache, this will never happen. 

But there is many more PSR6 cache libraries. I need to look at the specifications and not the implementations.